website/docs/r/autoscale_setting.html.markdown

```diff
@@ -0,0 +1,351 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_autoscale_setting"
+sidebar_current: "docs-azurerm-resource-autoscale-setting"
+description: |-
+  Create an autoscale setting.
```

Comment:
does this also covers other resources like Web Apps etc - or is it specific to VMSS?